todos-app/src/UUID.ts

```diff
@@ -0,0 +1,11 @@
+function setUUID() : string {
```

Comment:
maybe this function needs a better name, set usually implies that it's setting something, but here we are generating a new uuid

Reply:
Renamed it to GenerateUUID()